fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/mixin/resource/loader/ReloadableResourceManagerImplMixin.java

```diff
@@ -40,7 +43,18 @@
 	private ResourceType type;
 
 	@Inject(at = @At("HEAD"), method = "beginReloadInner")
-	public void reload(Executor var1, Executor var2, List<ResourceReloadListener> listeners, CompletableFuture future, CallbackInfoReturnable<CompletableFuture> info) {
+	private void reload(Executor var1, Executor var2, List<ResourceReloadListener> listeners, CompletableFuture future, CallbackInfoReturnable<CompletableFuture> info) {
 		ResourceManagerHelperImpl.sort(type, listeners);
 	}
+
+	@Inject(method = "method_29491", at = @At("HEAD"), cancellable = true)
```

Comment:
Could you throw this in a comment above so we know this is a synthetic method:

`private static synthetic method_29491(Ljava/util/List;)Ljava/lang/Object;`